tests/cases/py_client/lib_test.py

```diff
@@ -144,10 +173,10 @@ def testRestCore(self):
         flag = False
         try:
             self.client.getResource('user/badId')
-        except girder_client.HttpError as e:
-            self.assertEqual(e.status, 400)
-            self.assertEqual(e.method, 'GET')
-            resp = json.loads(e.responseText)
+        except requests.HTTPError as e:
+            self.assertEqual(e.response.status_code, 400)
+            self.assertEqual(e.request.method, 'GET')
```

Comment:
Since this has a `request` property, I assume this `HTTPError` was raised by `requests` natively? The Girder-raised exceptions don't pass a `request` argument to the `HTTPError` constructor.